subsys/bluetooth/controller/Kconfig

```diff
@@ -86,4 +86,39 @@ choice SOFTDEVICE_CONTROLLER_VARIANT
 
 endchoice
 
+if BT_CTLR_ECDH
+choice BT_CTLR_ECDH_LIB
+	prompt "SoftDevice Controller ECDH library"
+
+config BT_CTLR_ECDH_LIB_OBERON
+	select NRF_OBERON
+	bool "Use nrf_oberon SW crypto library for ECDH"
+
+config BT_CTLR_ECDH_LIB_TINYCRYPT
+	select TINYCRYPT
+	select TINYCRYPT_ECC_DH
+	bool "Use TinyCrypt library library for ECDH"
+
+endchoice
+
+config BT_CTLR_ECDH_STACK_SIZE
+	# NOTE: This value is derived from other symbols and should only be
+	# changed if required by architecture.
```

Comment:
Is this warning still needed ?
Value itself cannot be changed by the user anymore.

Reply:
Will remove this.